resolwe/process/tests/test_python_process.py

```diff
@@ -686,7 +718,7 @@ def test_python_process_iterate(self):
                 ]
             )
             # Assert iterate method was called the correct number of times.
-            self.assertEqual(calls, Process.objects.count() // chunk_size)
+            self.assertEqual(calls, math.ceil(Process.objects.count() / chunk_size))
```

Comment:
Why is this change necessary? If it fixes a bug it does belong into a new commit.